x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -193,6 +193,10 @@ export class HeadlessChromiumDriverFactory {
     // Puppeteer doesn't give a handle to the original ChildProcess object
     // See https://github.com/GoogleChrome/puppeteer/issues/1292#issuecomment-521470627
 
+    if (childProcess == null) {
+      throw new Error('TypeError! ChildProcess is null!');
```

Comment:
The tiniest of nits 🙈

```suggestion
      throw new TypeError('childProcess is null or undefined!');
```